### PR TITLE
Finding number of Digits in a Number

### PR DESCRIPTION
**Pull request by amino19**
#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1492"></a>

### CODE REVIEW

**Review by piyushpandey615 — CHANGES_REQUESTED** [review]
This code will only work for positive integers.

**Comment by amino19** [comment]
May this changes works!
How can I test this whole process in my local computer, so I can't get any rough mistakes & also smoothly goes on merge?
@Panquesito7

File: math/finding_number_of_Digits_in_a_Number.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * \details Basically, its a very basic math of
+ *  finding number of digits in a number (input),
+ *  For more details, refer Algorithms-Explanation
+ */
+
+
+/* Log based C++ program to count number of
+*  digits in a number/integer */
+
+#include <bits/stdc++.h>
```
Comment (cpyberry) [inline math/finding_number_of_Digits_in_a_Number.cpp:15]:
```suggestion
#include <iostream>
```

Please read [CONTRIBUTING.md](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md)
You cannot use `bits/stdc++.h`.

Comment (amino19) [reply math/finding_number_of_Digits_in_a_Number.cpp:15]: Okay, I forgot to read this; thanks for your suggestion, cpyberry! 😄

Comment (cpyberry) [reply math/finding_number_of_Digits_in_a_Number.cpp:15]: I am happy that I can help you :)

**Comment by Panquesito7** [comment]
> whole process in my local computer

I recommend using the [Gitpod Ready-to-Code](https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1492) service. There, you can run any commands you need to run as if you were in your computer but in the cloud. 🙂

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please fix [`clang-tidy`](https://github.com/TheAlgorithms/C-Plus-Plus/pull/1492/checks?check_run_id=2580311710) warnings.

**Comment by amino19** [comment]
Since my all test are passed, I've to close this Pull request, is little confusing..
I'll make a new one, by specifying all previous Pull Requests I made..

**Pull request closed without merging**